release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1614,6 +1614,38 @@ ERROR failed to fetch Master Machines: failed to load asset "Install Config": fa
 +
 As a workaround, if you have an installation configuration file, update it with a specific project id to use (`platform.gcp.projectID`). Otherwise, manually create an installation configuration file, and enter a specific project id. Run the installation program again, specifying the file. (link:https://issues.redhat.com/browse/OCPBUGS-15238[*OCPBUGS-15238*])
 
+[id="ocp-telco-ran-4-14-known-issues"]
+* During SR-IOV regression testing with two VRFs that have overlapping IP address ranges, the DHCP client container in one of the Pods gets stuck in a `ContainerCreating` state.
+It takes up to 7 minutes for the server and client Pods to receive IP allocations from the DHCP server.
++
+(link:https://issues.redhat.com/browse/OCPBUGS-18873[*OCPBUGS-18873*])
+
+* When the cluster has VRFs that have overlapping IP addresses on the primary network interface, traffic does get passed to secondary interfaces, both VRF and non-VRF.
+This error occurs with Intel E810 NICs. it does not occur with Mellanox Connect-X5 NICs.
++
+(link:https://issues.redhat.com/browse/OCPBUGS-19536[*OCPBUGS-19536*])
+
+* When a performance profile is applied to the cluster it creates a new `/ovs.slice` cgroup and moves OVS services under it.
```

Comment:
@yanirq Can you review/lgtm on this 4.14 known issue please?

Reply:
lgtm